boxes_post_processing.py

```diff
@@ -0,0 +1,249 @@
+import sys
+import os 
+import pandas as pd
+import time
+import numpy as np
+from gz.math7 import Quaterniond
```

Comment:
please update the README to include instructions about installing `gz-math7` python bindings